wled00/xml.cpp

```diff
@@ -186,6 +187,52 @@ void sappends(char stype, const char* key, char* val)
   }
 }
 
+void extractPin(JsonObject &obj, const char *key) {
```

Comment:
This is a good solution until we remake settings to read `cfg.json` directly.

Reply:
An possibly more lightweight alternative would be adding the pins currently allocated in `PinManager`, although I'm not sure if each usermod will actually allocate all pins it has configured in `cfg.json`, although that would likely be the case

Reply:
Talked about that withHenry Gabrielsky.